utils/utils.go

```diff
@@ -66,3 +66,8 @@ func AddAnnotation(obj metav1.Object, key string, value string) {
 func GetRegexMatcher(alerts []string) string {
 	return "^" + strings.Join(alerts, "$|^") + "$"
 }
+
+// GetRelableRegex returns a regex string from list of alerts and metrics
+func GetRelableRegex(alerts []string, metrics []string) string {
+	return ".*" + strings.Join(alerts, "|.*") + "|" + strings.Join(metrics, ";|") + ";"
+}
```

Comment:
- This func does not fit the criteria for the general util package. It is very specific and has a concrete and specific use case. 
- the implementation is wrong and will not produce the desired regexp. 

For a data set of 
```
var metrics = []string{
	"job:ceph_versions_running:count",
	"job:ceph_pools_iops_bytes:total",
}

var alerts = []string{
	"CephMdsMissingReplicas",
	"CephMgrIsAbsent",
}
```

This will result in 
```
.*CephMdsMissingReplicas|.*CephMgrIsAbsent|job:ceph_versions_running:count;|job:ceph_pools_iops_bytes:total;
```
where the desired state is
```
(ALERTS;(CephMdsMissingReplicas|CephMgrIsAbsent))|(job:ceph_versions_running:count;)|(job:ceph_pools_iops_bytes:total;)
```

I will suggest moving this func into the template file as 
```
func GetRelableRegex(alerts []string, metrics []string) string {
	return fmt.Sprintf(
		"(ALERTS;(%s))|%s",
		strings.Join(alerts, "|"),
		strings.Join(
			MapItems(metrics, func(str string) string { return fmt.Sprintf("(%s;)", str) }),
			"|",
		),
	)
}
```

And, of course, add the following util as a general util
```
func MapItems(source []string, transform func(string) string) []string {
	target := make([]string, len(source))
	for i := 0; i < len(source); i += 1 {
		target[i] = transform(source[i])
	}
	return target
}
```